docs/overview.md

```diff
@@ -51,8 +48,8 @@ be [exported](./export.md) in a variety of formats.
 
 ## Multi-Platform Support
 
-`jrnl` is compatible with most operating systems. Pre-compiled binaries are
-available through several distribution channels, and you can build from source.
+`jrnl` is compatible with most operating systems. You can download it using one
```

Comment:
Should this link to the installation page?

Reply:
Yes, definitely!